chrome/content/zotero/elements/itemPane.js

```diff
@@ -371,7 +364,7 @@
 		}
 
 		setReadLabel(isRead) {
-			var elem = document.getElementById('zotero-feed-item-toggleRead-button');
+			var elem = this.getCurrentPane().querySelector('#zotero-feed-item-toggleRead-button');
```

Comment:
But we shouldn't use `id` if we're creating this multiple places in the document — we should use a class.